### PR TITLE
Add conda installation to docs

### PR DESCRIPTION
**Pull request by A-CGray**
## Purpose

Now that CMPLXFOIL can be installed with conda I added a section explaining this in the docs

## Expected time until merged

3 seconds

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Checklist

- [ ] I have run `flake8` and `black` to make sure the Python code adheres to PEP-8 and is consistently formatted
- [ ] I have formatted the Fortran code with `fprettify` or C/C++ code with `clang-format` as applicable
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### CODE REVIEW

**Review by bernardopacini — DISMISSED** [review]
LGTM, thanks @A-CGray!

**Review by eytanadler — COMMENTED** [review]
Thanks for the update! What do you think about adding the line “conda install -c conda-forge cmplxfoil” directly into our docs (and maybe saying it supports Linux and macOS) to avoid another click? Sorry this review is a little late since it’s not within 3 sec.

**Comment by A-CGray** [comment]
> Thanks for the update! What do you think about adding the line “conda install -c conda-forge cmplxfoil” directly into our docs (and maybe saying it supports Linux and macOS) to avoid another click? Sorry this review is a little late since it’s not within 3 sec.

Could do, I thought I'd just link to the official conda forge page so we didn't have to worry about keeping something in our docs up to date with the correct installation instructions there, but maybe there is very little chance of the installation command changing?

File: doc/install.rst [hunk 1]
```diff
@@ -1,7 +1,10 @@
-Installation
-============
-CMPLXFOIL is a Python wrapper for the Fortran-based XFOIL code.
-Building and installing the code is a multi-step process that is machine specific.
+Conda Installation
```
Comment (eytanadler) [inline doc/install.rst:1]: Also just noticed, could you add
```
************
Installation
************
```
at the top of this page so it appears as a single entry in the left sidebar in the docs?

**Review by eytanadler — CHANGES_REQUESTED** [review]

**Review by eytanadler — APPROVED** [review]
LGTM!

**Comment by A-CGray** [comment]
Bump @bernardopacini

**Review by bernardopacini — APPROVED** [review]
Eek sorry, didn't realize my approval had been dismissed. Looks good to me.

**Pull request merged**